README.md

```diff
@@ -529,7 +364,13 @@ To upload to PyPI, use twine:
 python3 -m twine upload dist/*
 ```
 
-Details can be found in the [twine documentation](https://twine.readthedocs.io/en/stable/index.html).
+Details can be found in the [twine documentation](https://twine.readthedocs.io/en/stable/index.html). -->
+
+### Pre-Commit Hook
+
+Whenever you try to commit your changes (`git commit -m "my commit message"`), you may experience errors if your current shell doesn't have access to the dependencies required for the pre-commit hook. To remedy this, simply prefix your `git` command with `poetry run`. E.g.: `poetry run git commit -m "my commit message"`.
+
+If your code does not conform to formatting or style conventions, your commit will fail, and you will have to revise your code before committing it. Note, however, that our auto-formatter `black` does modify your files in-place when you run the pre-commit hook; you'll simply have to `git add` the changed files to stage the formatting changes, and you can attempt to commit again.
```

Comment:
Is this section still applicable, especially given the struck-through pre-commit part of the `Contributor Installation`?

Reply:
Good catch -- I meant to remove this.